components/env.rst

```diff
@@ -0,0 +1,114 @@
+.. index::
+   single: Env
+   single: Components; Env
+
+The Env Component
+=================
+
+    The Env Component parses ``.env`` files to make environment variables
+    stored in them accessible via ``getenv()``, ``$_ENV``, or ``$_SERVER``.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/env`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/env).
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+Sensitive information and environment-dependent settings should be defined as
+environment variables (as recommended for `twelve-factor applications
+<http://www.12factor.net/>`_. Using an `.env` file to store those environment
+variables eases development and CI management by keeping them in one "standard"
+place and agnostic of the technology stack you are using (Nginx vs PHP build-in
+server for instance).
+
+.. note::
+
+    PHP has a lot of different implementation of this "pattern". This
+    implementation goal is to replicate what ``source .env`` would do. So, it
+    tries to be as similar as possible with the default shells behavior, and
+    does not do anything not possible via ``source`` (like value validation).
+
+Usage
+-----
+
+First, require Symfony Env via Composer:
+
+.. code-block::
+
+    composer require symfony/env
+
+Load an ``.env`` file in your PHP application via ``Dotenv::load()``:
+
+.. code-block:: php
```

Comment:
In docs we are not allowed to use this `php` code block and we must the `::` trick.